code/modules/clothing/masks/gasmask.dm

```diff
@@ -106,6 +106,19 @@
 		to_chat(user, "<span class='notice'>Your Clown Mask has now morphed into [choice], all praise the Honkmother!</span>")
 		return 1
 
+/obj/item/clothing/mask/gas/cluwne_mask

```

Comment:
why not make this a subtype of a normal clown mask?

Reply:
Dont want to inherit the morphing of the clown mask. You will be jomker forever.